Networking/Networking/Model/Refund/OrderItemTaxRefund.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+
+
+/// Represents a Tax Refund for a specific Order Item.
+///
+public struct OrderItemTaxRefund: Codable {
```

Comment:
this struct seems to have the same content as `OrderItemTax`, do you know the differences between these two?

Reply:
ah nevermind! `OrderItemTaxRefund` is the tax for a refund vs. `OrderItemTax` is for an order

Reply:
I know, it's confusing. I tried thinking of better names and failed 😔

Reply:
no worries! the names are good, I was just confused for a little bit 😄